CHANGELOG.md

```diff
@@ -1,3 +1,69 @@
+## 1.10.0 (April 09, 2025)
+
+
+## 1.10.0 (April 08, 2025)
```

Comment:
Looks like there's a little mixup here.

Reply:
You're too fast :grin: Sorry I should have cherry-picked and pushed before opening the PR.